pkgs/os-specific/linux/msi-ec/default.nix

```diff
@@ -0,0 +1,52 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  linuxPackages,
+  kernel ? linuxPackages.kernel,
+  ...
+}:
+stdenv.mkDerivation rec {
+  pname = "msi-ec-kmods";
+  version = "0.08";
+
+  src = fetchFromGitHub {
+    owner = "m1dugh";
+    repo = "msi-ec";
+    rev = "60aad95e2c8948140232f4c4d70f4ceba821d5a5";
+    hash = "sha256-qgM4KeGoLvbVsii5YR3GTGi2MVlU+JeArzEoA+6A3a0=";
+  };
```

Comment:
Is there a reason this is using your fork? From what I can tell, upstream is still maintained and there are no technical changes in your  fork.

Reply:
There is still a PR running on the original repo, I intend to change it as soon as the PR is approved

Reply:
I'm hesitant to use a fork, especially since the upstream appears to be maintained and [the upstream PR in question which originates from this fork](https://github.com/BeardOverflow/msi-ec/pull/162) has some unresolved change requests from the maintainer.